src/MeetingsSDKAdapter.js

```diff
@@ -259,8 +273,8 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     });
 
     const muteEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_MUTE).pipe(
-      filter((event) => event.control === MUTE_AUDIO_CONTROL),
-      map(() => ({...muted, state: MeetingControlState.INACTIVE}))
+      filter((event) => event.control === AUDIO_CONTROL),
+      map(({state}) => (state ? muted : unmuted))
```

Comment:
Where does this state come from?

Reply:
so this comes from line 234, where we are emitting the event with the given object which contains `state` prop

Reply:
basically we have two listeners, one in `getMeeting` and one in `audioControl` and they both listen to the changes emitted from `hanldeLocalAudio` method

Reply:
Ok, so it is from our constructed sdk event. I didn't remember seeing the `state` property on the native sdk events.